src/planactions.ts

```diff
@@ -42,3 +43,14 @@ export const submitPlan: ActionCreator<ThunkAction<Promise<AnyAction>, IPlanSubm
         });
     };
 };
+
+export const clearQueue: ActionCreator<ThunkAction<Promise<AnyAction>, null, null, any>> = () => {
+    return async (dispatch: Dispatch) => {
+        dispatch(loading());
+        const queueState = await clearQueueAPI();
+        return dispatch({
+            queueState,
+            type: PlanActionTypes.GETPLANLIST
+        });
+    };
+};
```

Comment:
Add a newline at the end of the file to keep git happy.

Reply:
Done.